spec/models/book_spec.rb

```diff
@@ -0,0 +1,94 @@
+require 'rails_helper'
+
+describe Book, type: :model do
+  it do
+    should validate_presence_of(:genre)
+  end
+  it do
+    should validate_presence_of(:author)
+  end
+  it do
+    should validate_presence_of(:image)
+  end
+  it do
+    should validate_presence_of(:title)
+  end
+  it do
+    should validate_presence_of(:publisher)
+  end
+  it do
+    should validate_presence_of(:year)
+  end
+
+  subject(:book) do
```

Comment:
We use factory girl for creating objects.
This should be

subject(:book) { create(:book) }